### DISCUSSION TITLE
¿Cómo agregar varios usuarios a un prellenado de users? (PBSendSurvey)

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#394] [2. Questions and Answers] [answered]
Buenas,

Para una implementación necesito enviar a un formulario un listado de usuarios prellenados con la etapa PBSendSurvey.
¿En qué formato debe estar lo que se le envía para prellenar múltiples usuarios?

Buena semana. 
Saludos!

**Comment by acedward** [comment, marked as answer]
Por el momento no es posible con PBSendSurvey v.3.0.0

**Reply by seiturra** [reply]
Para solucionar el requerimiento ¿Podemos agregar la función string.split a cotLang?

$VALUE#name: "hola - mundo"
$VALUE#name|[split=> - ]
retorna el array: [hola, mundo]

**Reply by seiturra** [reply]
Lo solucioné con un network request distinto a lo que había pensado inicialmente
"name" : "NWRequest", 
"data" : {
    "url" : "$JOIN#/#($ENV#BASEURL)#api#users#find", 
    "method" : "POST", 
    "defaultAuth" : true, 
    "body" : "($JOIN##\`\`\`{\"extra.PUESTO_TRABAJO\":{\"$in\":[\"\`\`\`#($VALUE#sentAnswer|data|[find=>identifier=edsnew_work_position]|process|0)#\`\`\`\"]}}\`\`\`)|[json=>parse]"
},
